kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/MessagesTable.tsx

```diff
@@ -19,6 +19,7 @@ const MessagesTable: React.FC<MessagesTableProp> = ({ messages, onNext }) => {
       <table className="table is-fullwidth">
         <thead>
           <tr>
+            <th>Key</th>
```

Comment:
since the key column is sometimes empty, I suggest moving it to the second position. After timestamp. cc: @germanosin

Reply:
done